multipart-uploader/multipart-uploader-s3/src/main/java/org/apache/celeborn/S3MultipartUploadHandler.java

```diff
@@ -66,6 +67,9 @@ public class S3MultipartUploadHandler implements MultipartUploadHandler {
 
   private Integer s3MultiplePartUploadMaxRetries;
 
+  long initBackoff = 100; // 100ms
```

Comment:
Adding new configs for these parameters will be better.

Reply:
Ok, but it means that we should also change the usage of the retry policy here https://github.com/apache/celeborn/blob/5f58fb1e3e0fecc721db4e38292b3f6d2624a1af/multipart-uploader/multipart-uploader-s3/src/main/java/org/apache/celeborn/S3MultipartUploadHandler.java#L85 to ensure we set same config here than the one that will be set on the celeborn config

Reply:
Yes, we should also change the module of the Multipart upload handler. A complete change PR will be better.